src/app/recipe-creator/recipe-creator.component.ts

```diff
@@ -23,21 +24,53 @@ export class RecipeCreatorComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  selectedFile1: File = null;
+  picture1: string
+  selectedFile2: File = null;
+  picture2: string
+
+  handleUpload1(event) {
+    const file = event.target.files[0];
+    const reader = new FileReader();
+    this.selectedFile1 = <File>event.target.files[0];
+    reader.readAsDataURL(file);
+    reader.onload = () => {
+      console.log(reader.result);
+      this.picture1 = reader.result.toString();
+    };
+  }
+  handleUpload2(event) {
+    const file = event.target.files[0];
+    const reader = new FileReader();
+    this.selectedFile2 = <File>event.target.files[0];
+    reader.readAsDataURL(file);
+    reader.onload = () => {
+      console.log(reader.result);
+      this.picture2 = reader.result.toString();
+    };
+  }
+
```

Comment:
What if the user wants to add more images?

Reply:
I agree with you on that. I would also rather make it with a list. But I was running out of time.